pkg/asset/machines/worker.go

```diff
@@ -26,9 +26,7 @@ import (
 )
 
 func defaultAWSMachinePoolPlatform() awstypes.MachinePool {
-	return awstypes.MachinePool{
-		InstanceType: "m4.large",
-	}
+	return awstypes.MachinePool{}
```

Comment:
why is this emptied ?

Reply:
@wking ^

Reply:
> why is this emptied ?

Because we no longer have any default AWS configuration shared by both masters and workers.  They each have their own defaults for instance types (and, with #1079) for volumes, that get applied directly after `defaultAWSMachinePoolPlatform()` calls.  In fact, my preference would be to drop `defaultAWSMachinePoolPlatform` entirely in favor of an explicit `awstypes.MachinePool{}` for initializing AWS machine pools.

Reply:
Ah, maybe this was working around #1076.